common/goods.txt

```diff
@@ -87,26 +87,26 @@ consumer_goods = {
 		cost = 0.6
 		color = { 216 185 77 }
 	}
-#	literature = {
-#		cost = 1
-#		color = { 0 0 0 }
-#	}
+	literature = {
```

Comment:
We'll want to update the description to the introduction event (29000) to include the new goods